Marlin/language_da.h

```diff
@@ -157,6 +157,15 @@
 #define MSG_BED_DONE                        "Plade opvarmet"
 #define MSG_ENDSTOP_ABORT                   "Endstop abort"
 
+#define MSG_BED_Z                           "Plade Z"
+#define MSG_HEATING_FAILED_LCD              "Opvarmning mislykkedes"
+#define MSG_ERR_REDUNDANT_TEMP              "Fejl: reserve temperatur"
+#define MSG_THERMAL_RUNAWAY                 "Temperatur løber løbsk"
+#define MSG_ERR_MAXTEMP                     "Fejl: Maks temperatur"
+#define MSG_ERR_MINTEMP                     "Fejl: Min temperatur"
+#define MSG_ERR_MAXTEMP_BED                 "Fejl: Maks temperatur plade"
```

Comment:
Instead of…
```
#define MSG_ERR_MAXTEMP_BED                 "Fejl: Maks temperatur plade"
16: "Fejl: Maks tempe"
20: "Fejl: Maks temperatu"
22: "Fejl: Maks temperatur "
```
…maybe…
```
#define MSG_ERR_MAXTEMP_BED                 "Fejl: Maks P temp"
16: "Fejl: Maks P tem"
20: "Fejl: Maks P temp"
22: "Fejl: Maks P temp"
```